### PR TITLE
js file for creating trips

### PR DESCRIPTION
**Pull request by maleny25**
File that saves trips and the trip id to a user

### CODE REVIEW

File: src/javascript/createTrip.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        location: form.location.value,
+        date_from: form.date_from.value,
+        date_to: form.date_to.value,
+        description: form.description.value,
+        events: []
+    })
+    .then((docRef) => {
+    docid = docRef.id;
+    var tripdoc = db.collection('trips').doc(docRef.id);
+    var setWithMerge = tripdoc.update({
```
Comment (thechungster) [inline src/javascript/createTrip.js:21]: nit :tabbing

In general, you can use an autoformatter for things like this. The ide you're using should have one built in, if not there's extensions you can get to autoformat (such as https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)

File: src/javascript/createTrip.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      code: docRef.id
+    });
+    // add trip to user's list
```
Comment (thechungster) [inline src/javascript/createTrip.js:24]: You can delete these commented out lines if they're not needed anymore

File: src/javascript/createTrip.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    // get user id
+    // var userid = firebase.auth().currentUser;
+    // get user's list of trips to update them
+    // var user = db.collection('users').doc(userid);
+    var user = db.collection('users').doc(userid);
+    user.update({
+    trips: firebase.firestore.FieldValue.arrayUnion(docRef.id)
+    });
+    //clear form
+    form.name.value = '';
```
Comment (thechungster) [inline src/javascript/createTrip.js:34]: Does `form.reset()` work? 
https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/reset

File: src/javascript/createTrip.js (hunk 1 above)
Comment (maleny25) [reply src/javascript/createTrip.js:25]: done

File: src/javascript/createTrip.js (hunk 2 above)
Comment (maleny25) [reply src/javascript/createTrip.js:28]: removed

File: src/javascript/createTrip.js (hunk 3 above)
Comment (maleny25) [reply src/javascript/createTrip.js]: changed to form.reset()

**Comment by maleny25** [comment]
DON'T MERGE THIS BRANCH

**Pull request closed without merging**